onlyOffice/src/main/java/org/exoplatform/platform/qa/ui/onlyOffice/pageobject/OnlyOfficeEditingPage.java

```diff
@@ -30,13 +30,12 @@ public OnlyOfficeEditingPage(TestBase testBase) {
     /**
      * Check that document is opened on OnlyOffice Editor
      * @param documentName
-     * @param userName
+     * @param extension
      */
-    public void checkOpeningDocumentWithEditOnline(String documentName, String userName) {
-        $(byText(documentName + ".docx")).waitUntil(visible, openBrowserTimeoutMs);
+    public void checkOpeningDocumentWithEditOnline(String documentName,String extension) {
```

Comment:
Why did you split the doc name ? It seems the extension is never use alone.

Reply:
for "checkButtonDocument(document)" I use only the name of document for US1 and US2, and I think it's more clear for all types